fe/fe-core/src/main/java/org/apache/doris/planner/SetOperationNode.java

```diff
@@ -283,14 +283,12 @@ private boolean isChildPassthrough(
             if (childSlotRef == null) {
                 return false;
             }
-            if (VectorizedUtil.isVectorized()) {
-                if (childSlotRef.getDesc().getSlotOffset() != setOpSlotRef.getDesc().getSlotOffset()) {
-                    return false;
-                }
-            } else {
-                if (!childSlotRef.getDesc().layoutEquals(setOpSlotRef.getDesc())) {
-                    return false;
-                }
+            if (VectorizedUtil.isVectorized()
+                    && childSlotRef.getDesc().getSlotOffset() != setOpSlotRef.getDesc().getSlotOffset()) {
```

Comment:
maybe it it better to only check type and nullable when use vectorized engine, then we have more chance to do passthrough